PublicOnboardingDemo/clients/LandingPage/controllers/client/onStart.js

```diff
@@ -0,0 +1,32 @@
+    
+    var isPublic = client.isPublic();
+    
+    var intro = client.getWidget("WelcomeLine");
+    intro.text = isPublic ? "Welcome to VANTIQ! You are currently in Public Mode." : "Welcome to VANTIQ! You are currently in Private Mode.";
+    
+    var uuid = client.getUsername();
+    var username = client.getUserRecord();
```

Comment:
I don't think you need to call this now if you're going to call it in the function below. Also, this is the full user record, not just their username, so the variable should be renamed.